friends.html

```diff
@@ -1 +1,177 @@
+            <div class="col-md-3 unfriend">
+              <button>Unfriend me</button>
+            </div>
+          </div>
+          <div class="row friend-tile">
+            <div class="col-md-3">
+              <a href="#">
+                <img class="img-thumbnail" src="img/user_silhouette_generic.gif">
+              </a>
+            </div>
+            <div class="col-md-6 friend-name">
+              <a href="#">Hermione Granger</a>
+              <p class="friend-count">432 friends</p>
+            </div>
+            <div class="col-md-3 unfriend">
+              <button>Unfriend me</button>
+            </div>
+          </div>
+          <div class="row friend-tile">
+            <div class="col-md-3">
+              <a href="#">
+                <img class="img-thumbnail" src="img/user_silhouette_generic.gif">
+              </a>            </div>
+            <div class="col-md-6 friend-name">
+              <a href="#">Hermione Granger</a>
+              <p class="friend-count">432 friends</p>
+            </div>
+            <div class="col-md-3 unfriend">
+              <button>Unfriend me</button>
+            </div>
+          </div>
+        </div>
+        <!-- right side of friends section -->
+        <div class="col-md-6">
+          <div class="row friend-tile">
+            <div class="col-md-3">
+              <a href="#">
+                <img class="img-thumbnail" src="img/user_silhouette_generic.gif">
+              </a>            </div>
+            <div class="col-md-6 friend-name">
+              <a href="#">Hermione Granger</a>
+              <p class="friend-count">432 friends</p>
+            </div>
+            <div class="col-md-3 unfriend">
+              <button>Unfriend me</button>
+            </div>
+          </div>
+          <div class="row friend-tile">
+            <div class="col-md-3">
+              <a href="#">
+                <img class="img-thumbnail" src="img/user_silhouette_generic.gif">
+              </a>            </div>
+            <div class="col-md-6 friend-name">
+              <a href="#">Hermione Granger</a>
+              <p class="friend-count">432 friends</p>
+            </div>
+            <div class="col-md-3 unfriend">
+              <button>Unfriend me</button>
+            </div>
+          </div>
+          <div class="row friend-tile">
+            <div class="col-md-3">
+              <a href="#">
+                <img class="img-thumbnail" src="img/user_silhouette_generic.gif">
+              </a>            </div>
+            <div class="col-md-6 friend-name">
+              <a href="#">Hermione Granger</a>
+              <p class="friend-count">432 friends</p>
+            </div>
+            <div class="col-md-3 unfriend">
+              <button>Unfriend me</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
```

Comment:
I really appreciate the CSS classes used here. They described the content perfectly, and were used sparingly.